server/services/notify/notifysubscriptions/subscriptions_backend.go

```diff
@@ -179,6 +183,10 @@ func (b *Backend) OnMention(userID string, evt notify.BlockChangeEvent) {
 		return
 	}
 
+	// TODO: Automatically add user to board?  Fail and show UI?  Waiting for PM decision.
```

Comment:
@chenilim @wuwinson we should make a decision here, for consistency with Mattermost Channels, we should ask the user in the frontend for that, if the user has the right permissions to do it.